src/mean.jl

```diff
@@ -29,99 +9,139 @@ julia> srs = SimpleRandomSample(apisrs;popsize=:fpc);
 
 julia> mean(:enroll, srs)
 1×2 DataFrame
- Row │ mean     sem     
+ Row │ mean     SE     
      │ Float64  Float64 
 ─────┼──────────────────
    1 │  584.61  27.3684
 ```
 """
 function mean(x::Symbol, design::SimpleRandomSample)
+    function se(x::Symbol, design::SimpleRandomSample)
```

Comment:
I disagree with the inner functions. I suggest having external functions that we don't export and start with an `_` (`_se`)

Reply:
Users can still access external functions even if they aren't exported. For example, `Survey.se`. Also, it's clear that which function helps which one. 

Inner functions have been allowed in Julia for this reason.